assets/css/style.css

```diff
@@ -298,3 +310,90 @@ ul.my_work_do li {
 
   letter-spacing: 2px;
 }
+
+@media (max-width: 770px) {
```

Comment:
[common breakpoints](https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/)